messages/dashboard.json

```
@@ -1,4 +1,16 @@
 [
+  {
+    "message": "Enjoying rotki? Consider ",
+    "message_highlight": "supporting us in",
+    "action": {
+      "text": "Octant Epoch 5",
+      "url": "https://octant.app/home"
+    },
+    "period": {
+      "start": 1728835200,
+      "end": 1729987200
```

Should also end same time, so 16:00 UTC: 
```suggestion
      "end": 1730044800
```